website/docs/docs/build/udfs.md

```diff
@@ -107,52 +108,57 @@ To define UDFs in dbt, refer to the following steps:
     <TabItem value="Snowflake">
     ```sql
     CREATE OR REPLACE FUNCTION udf_db.udf_schema.is_positive_int(a_string STRING)
-    RETURNS BOOLEAN
+    RETURNS INTEGER
     LANGUAGE SQL
+    IMMUTABLE
     AS $$
-      REGEXP_CONTAINS(a_string, r'^[0-9]+$')
+      REGEXP_INSTR(a_string, '^[0-9]+$')
     $$;
     ```
+
     </TabItem>
 
     <TabItem value="Redshift">
     ```sql
-    CREATE OR REPLACE FUNCTION udf_db.udf_schema.is_positive_int(VARCHAR)
-    RETURNS BOOLEAN
-    VOLATILE # Technically this function could be set as STABLE, but we don't support setting volatility yet
+    CREATE OR REPLACE FUNCTION udf_schema.is_positive_int(a_string VARCHAR)
+    RETURNS INTEGER
+    IMMUTABLE
     AS $$
-      REGEXP_CONTAINS($1, r'^[0-9]+$')
+      SELECT REGEXP_INSTR(a_string, '^[0-9]+$')
     $$ LANGUAGE SQL;
     ```
     </TabItem>
 
     <TabItem value="BigQuery">
     ```sql
     CREATE OR REPLACE FUNCTION udf_db.udf_schema.is_positive_int(a_string STRING)
-    RETURNS BOOLEAN
+    RETURNS INT64
     AS (
-      REGEXP_CONTAINS(a_string, r'^[0-9]+$')
+      REGEXP_INSTR(a_string, r'^[0-9]+$')
     );
+
     ```
     </TabItem>
 
     <TabItem value="Databricks">
     ```sql
     CREATE OR REPLACE FUNCTION udf_db.udf_schema.is_positive_int(a_string STRING)
-    RETURNS BOOLEAN
-    LANGUAGE SQL
-    RETURN
-      REGEXP_CONTAINS(a_string, r'^[0-9]+$');
+    RETURNS INT
+    DETERMINISTIC
+    RETURN regexp_instr(a_string, '^[0-9]+$');
     ```
     </TabItem>
 
     <TabItem value="Postgres">
+
     ```sql
-    CREATE OR REPLACE FUNCTION udf_db.udf_schema.is_positive_int(a_string STRING)
-    RETURNS BOOLEAN
+    CREATE OR REPLACE FUNCTION udf_schema.is_positive_int_pos(a_string text)
+    RETURNS int
+    LANGUAGE sql
+    IMMUTABLE
     AS $$
-      REGEXP_CONTAINS(a_string, r'^[0-9]+$')
-    $$ LANGUAGE SQL;
+      SELECT regexp_instr(a_string, '^[0-9]+$')
```

Comment:
uppercase `REGEXP_INSTR`?

Reply:
wouldn't that just be whatever you had put in your `functions/is_positive_int.sql` file? in the example above that's uppercase